daemon/openrazer_daemon/dbus_services/dbus_methods/argb_controller.py

```diff
@@ -0,0 +1,317 @@
+from openrazer_daemon.dbus_services import endpoint
+
+
+def _get_channel_brightness(self, channel):
+    driver_path = self.get_driver_path(channel + '_led_brightness')
+    with open(driver_path, 'r') as driver_file:
+        return float(driver_file.read().strip()) / (255.0 / 100.0)
+
+
+@endpoint('razer.device.lighting.channel1', 'getChannel1Brightness', out_sig='d')
+def get_channel1_brightness(self):
```

Comment:
Why not a method `get_channel_brightness(self, channel)`? Then we don't need to copy paste each method 6 times